Server/fixtures/mobile.js

```diff
@@ -31,39 +31,63 @@ exports.createFixtures = async () => {
       data: {
         id: 1,
         city: "Nantes",
-        address: "Rue d'Alger",
-        latitude: 47.210537, // Epitech Nantes
-        longitude: -1.566808,
+        address: "Rue George",
+        latitude: 47.218371,
+        longitude: -1.553621,
         saveName: "container",
         containerMapping: "",
-        price: 500.5,
+        price: 0,
         width: 12,
         height: 5,
         items: {
           create: [
             {
               id: 1,
-              name: "Ballon de volley",
-              price: 0.5,
+              name: "ballon de volley",
```

Comment:
maybe put back the uppercase at the start

Reply:
I put back